VisualRadio/app/src/main/java/altus/visualradio/MainListingActivity.java

```diff
@@ -0,0 +1,142 @@
+package altus.visualradio;
+
+import android.app.Activity;
+import android.app.ListActivity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.ListView;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import altus.visualradio.models.ListDetailSetter;
+
+
+public class MainListingActivity extends ListActivity {
+    private static      String index_filename = "VS_index_feed.json";
+    private             ArrayList<ListDetailSetter> index_detail_setter;
+    private             JSONObject json_object;
+
+    private static      String JSON_INDEX_KEY = "com.index_feed";
+    public static        String TITLE_KEY(){return "com.visual.header";}
```

Comment:
I was also unsure. However as it was also a unique key value being returned, I decided to keep the naming convention in the line with the other key values.